changelog.txt

```diff
@@ -2,3 +2,5 @@ fixed - Fixes a bug in Cloud Firestore emulator where rulesets with no-op writes
 fixed - Reduced lock contention in Cloud Firestore emulator during concurrent writes to a document.
 feature - Serving Hosting locally can now proxy to the live Cloud Run service.
 fixed - Fixes a bug where fields in Cloud Firestore indexes would be sorted alphabetically for display.
+fixed - Fixes a bug for Firestore REST API.
```

Comment:
please specify that this is for the emulator

Reply:
and let's add a bit more detail here, this was an issue where the REST api would yield 404s to root collections